app/controllers/providers/application_confirmations_controller.rb

```diff
@@ -1,5 +1,7 @@
 module Providers
   class ApplicationConfirmationsController < ProviderBaseController
-    def show; end
+    def show
+      legal_aid_application.update!(provider_step: :check_provider_answers)
```

Comment:
The provider step is updated as a before action (see `ApplicationDependable`, which is included into `ProviderBaseController`). So with your code here `provider_step` will be written to twice in this action. A better solution would be to overwrite the `set_legal_aid_application` method in this controller:
```ruby
def show; end

private

def set_legal_aid_application
  legal_aid_application.update!(provider_step: :check_provider_answers)
end
```

Reply:
Thanks, Implementing it now. 👍